### ISSUE TITLE
Not working for Tumblr accounts in comments

### ISSUE THREAD

**Issue by NoneTheFewer** [shinigami-eyes/shinigami-eyes#84]
I'm using version 1.0.30 of the Firefox add-on.

Steps:

* Find any post with comments (not just reblogs) and expand the comments.
* Right click on the account name for any commenter and select Shinigami Eyes > "Mark as anti-trans".

Alas:

<img width="593" alt="image" src="https://user-images.githubusercontent.com/109307890/190655774-83752709-60b2-4444-a02b-3a7ef702f3ef.png">

This is the `A` element around the username, using `@staff` as an example:

```<a target="_blank" rel="noopener" href="/staff" role="link" class="tNyYL BSUG4" data-login-wall-type="blogView" data-login-wall-blog-name="staff" data-login-wall-redirect="/dashboard" tabindex="0" shinigami-eyes-link-id="7">staff</a>```

**Comment by citrusella** [comment]
A workaround that works for any blog which is not login-only is to mouse over and right-click their name in the little "card" thing that pops up on hover, where the url is a name.tumblr.com one.

**Comment by NoMoreStars** [comment]
That workaround isn't working for me, none of the titles I hover over are currently using the "name.tumblr.com" format but instead "tumblr.com/name"

**Comment by citrusella** [comment]
Yeah, it was working 11 days ago but I noticed it didn't seem to work later. I think I saw *some*thing else that had the .tumblr.com structure but I don't remember what.

**Comment by XtinaSchelin** [comment]
Tumblr rolled out some changes at some point that increase the odds of finding tumblr[dot]com/[username] links, and this means (a) I don't see the red/green bordering and (b) I can't add it. (Thanks, Tumblr, as always.)
